upup/models/cloudup/_aws/topologies/_topology_private/bastion.yaml

```diff
@@ -116,23 +126,16 @@ autoscalingGroup/bastion.{{ ClusterName }}:
     Name: bastion-{{ GetBastionZone }}.{{ ClusterName }}
     KubernetesCluster: {{ ClusterName }}
 
-# ---------------------------------------------------------------
-# TODO Kris
-#
-# So as it stands we are *NOT* defining a friendly CNAME for the
-# bastion ELB. I think this is a good thing.
-#
-# If we came up with a formula EG: bastion.<clusterName> we could
-# be exposing ourselves to a few threats :)
-#
-# I think it's best in this situation to err on the side of
-# caution and force the end user to define something convenient
-# on their own.
-#
-# TLDR; If you want a friendly CNAME for your bastion - you have
-# to build it yourself. Kops won't support that
-#
-# Right?
-# ---------------------------------------------------------------
-
+{{ if IsBastionDNS }}
```

Comment:
This if statement is fine for now (as long as the comments don't give us any trouble!) 

But I would like a PR out later when we refactor the master tags to squeeze this in as a tag. I will probably code it, but just documenting here for good measure. https://github.com/kubernetes/kops/issues/835

No change needed.